spec/unit/plugins/darwin/filesystem2_spec.rb

```diff
@@ -0,0 +1,96 @@
+#
+# Author:: Phil Dibowitz (<phil@ipom.com>)
+# Copyright:: Copyright (c) 2015 Facebook, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require File.expand_path(File.dirname(__FILE__) + '/../../../spec_helper.rb')
+
+describe Ohai::System, "darwin filesystem2 plugin" do
+  before(:each) do
+    @plugin = get_plugin("darwin/filesystem2")
```

Comment:
That makes sense.
However, almost nothing else in the tree does this (almost everything has plugin be a classvar) - and this is a mostly-copy of the linux version which works the way this one does (on a classvar). Rather than change all this away from that, I'd rather get it in as-is... if there's not a strong objection.